python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -53,7 +49,7 @@ def _is_fastai_model(model):
     return str(type(model)).endswith(FASTAI_MODEL_SUFFIX)
 
 
-def _wrap_image_model(model, examples, model_task, is_function):
+def _wrap_image_model(model, examples, model_task, is_function, num_classes=None):
```

Comment:
Consider same arg name between `num_classes` in this function and `number_of_classes` in `WrappedObjectDetectionModel`. If they have different meaning, can perhaps tweak naming and/or clarify in the docstrings.